src/Exercise5/numberToBinary.js

```diff
@@ -0,0 +1,8 @@
+function numberToBinary(num) {
+  if (typeof num !== "number") {
+    throw new Error(`${num} is not a number`);
+  }
+  return num < 0
+    ? `${num}² is 1${num.toString(2).replace("-", "")}`
+    : `${num}² is ${num.toString(2)}`;
+}
```

Comment:
Here instead of returning that whole string, you can return only the binary format, because if you were to use that function in another function or somewhere else, it would take an extra lines of code to get the real value